Makefile

```diff
@@ -4,6 +4,11 @@ ALL_SRC := $(shell find . -name '*.go' \
                                 -not -path './testbed/*' \
                                 -type f | sort)
 
+# All source code and documents. Used in spell check.
+ALL_SRC_AND_DOC := $(shell find . \( -name "*.md" -o -name "*.go" \) \
```

Comment:
Can it do yaml files too? We have comments on those.

Reply:
Yes, added yaml files.